packages/schematics/angular/utility/latest-versions.ts

```diff
@@ -12,6 +12,7 @@ export const latestVersions = {
   RxJs: '~6.3.3',
   ZoneJs: '~0.8.26',
   TypeScript: '~3.1.1',
+  TsLib: '^1.9.0',
   // The versions below must be manually updated when making a new devkit release.
   DevkitBuildAngular: '~0.9.0-rc.1',
   DevkitBuildNgPackagr: '~0.9.0-rc.1',
```

Comment:
It seems like Angular and DevkitBuild* here need to be updated as well? Perhaps in a separate PR, but they seem quite out of date and are this way on `master` too.